api.go

```diff
@@ -126,6 +139,7 @@ func (api *API) request(endpoint string, method string, params map[string]interf
 		if api.retryCount < MAX_RETRIES {
 			api.retryCount = api.retryCount + 1
 			b := api.backoff.Duration()
+			log.Global().WithField("backoff duration values", b).WithField("calls made ", calls).WithField("calls limit ", total).Error("time to sleep")
```

Comment:
Yeah, would be good not to rely on private packages so that anyone can use this package. I'm okay with this in the short term if you want.